spec/fluent.asdl

```diff
@@ -73,10 +72,10 @@ module Fluent
            | StringExpression(string)
     
     iden = Identifier(str name, span? span)
-    symb = Symbol(str name, span? span)
+    varname = VariantName(str name, span? span)
     fun = Function(str name, span? span)
     string = StringExpression(str value, span? span)
     number = NumberExpression(str value, span? span)
 
-    comment = Comment(str content, span? span)
+    comment = Comment(int level, str content, span? span)
```

Comment:
Would it better to introduce three node types instead of the `level` field? E.g. `SingleComment`, `DoubleComment` and `TripleComment`.

Reply:
Not sure which way to go. The concern I have is about semantically correct mixed level comments:

```properties
# Foo
### Bar
# Baz
## Qux
key = Value
```

Reply:
These would be four independent `Comment` entries with levels: `1`, `3`, `1`, and `2`. Or four independent entries: `SingleComment`, `TripleComment`, `SingleComment`, and a `DoubleComment`.

Reply:
> Would it better to introduce three node types instead of the level field? E.g. `SingleComment`, `DoubleComment` and `TripleComment`.

@Pike, do you have an opinion or a recommendation about this?

Reply:
I wouldn't call 'em by markup, but by purpose. So, `FileComment`, `GroupComment`, `MessageComment` or just `Comment`. Or just leave them with levels.

Reply:
I went for `Comment`, `GroupComment` and `FileComment`. They are explicit and clearly communicate the role. They also define the interpretation of each of the comment types right at the level of the syntax which will will improve the portability of FTL files.